src/AvaloniaMath/TexFormulaParser.cs

```diff
@@ -363,80 +529,108 @@ private static SourceSpan ReadElement(SourceSpan value, ref int position)
                         if (value.Length > position && value[position] == leftBracketChar)
                         {
                             // Degree of radical is specified.
-                            degreeFormula = this.Parse(
+                            degreeFormula = Parse(
                                 ReadElementGroup(value, ref position, leftBracketChar, rightBracketChar),
-                                formula.TextStyle);
+                                formula.TextStyle,
+                                environment.CreateChildEnvironment());
                         }
 
-                        var sqrtFormula = this.Parse(ReadElement(value, ref position), formula.TextStyle);
+                        var sqrtFormula = this.Parse(
+                            ReadElement(value, ref position),
+                            formula.TextStyle,
+                            environment.CreateChildEnvironment());
 
                         source = value.Segment(start, position - start);
-                        return new Radical(source, sqrtFormula.RootAtom, degreeFormula?.RootAtom);
+                        return new Tuple<AtomAppendMode, Atom>(
+                            AtomAppendMode.Add,
+                            new Radical(source, sqrtFormula.RootAtom, degreeFormula?.RootAtom));
                     }
+                /* TODO: What?
                 case "underline":
```

Comment:
It appeared in #145, but now not in master

Reply:
It's here https://github.com/ForNeVeR/wpf-math/blob/0ed0a3b1c32a0ac5784ec6747cbd454242b6d592/src/WpfMath/Parsers/StandardCommands.cs#L105